examples/rectify_image.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+
+import cameramodels
+import cv2
+import numpy as np
+import open3d
+
+from cameramodels.align import align_depth_to_rgb
+
+
+rotation_matrix = np.array(
+    [[1, 0, 0, 0],
+     [0, -1, 0, 0],
+     [0, 0, -1, 0],
+     [0, 0, 0, 1]],
+    dtype=np.float32)
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Align Example')
+    parser.add_argument("--no-rgb-rect", action='store_true')
+    parser.add_argument("--no-depth-rect", action='store_true')
+    args = parser.parse_args()
+
+    depth_to_rgb_transform = np.load(
+        './data/depth_to_rgb_transformation.npy')
+
+    depth_cameramodel = cameramodels.PinholeCameraModel.from_yaml_file(
+        './data/k4a_depth_camera_info.yaml')
+    bgr_cameramodel = cameramodels.PinholeCameraModel.from_yaml_file(
+        './data/k4a_rgb_camera_info.yaml')
+    depth_img = cv2.imread('./data/k4a_depth_image_raw.png',
+                           cv2.IMREAD_ANYDEPTH)
```

Comment:
How about writing like
```
current_dir = osp.dirname(osp.abspath(__file__))
depth_to_rgb_transform = np.load(
    osp.join(current_dir, 'data/depth_to_rgb_transformation.npy'))
```
to run from outside the example directory?

Reply:
I agree.
Thanks.

Reply:
OK, Thanks